enterprise_subsidy/apps/api/v1/views/transaction.py

```diff
@@ -386,11 +389,17 @@ def create(self, request, *args, **kwargs):
 
         Returns:
             rest_framework.response.Response:
-                400: If there are missing or otherwise invalid input parameters.  Response body is JSON with a single
+                400: There are missing or otherwise invalid input parameters.  Response body is JSON with a single
+                     `Error` key.
+                403: The requester has insufficient create permissions.  Response body is JSON with a single `Error`
+                     key.
+                422: The subisdy is not redeemable in a way that IS NOT retryable (e.g. the balance is too low, or
+                     content is not in catalog, etc.).  Response body is JSON with a
+                     single `Error` key.
+                429: The subisdy is not redeemable in a way that IS retryable (e.g. something else is already holding a
```

Comment:
My reservation about using 422 was mainly that we could distinguish between retryable vs. non-retryable by treating 422 as non-retryable and using something else for retryable.

My reservation about using 429 was that same statement from mozilla claiming that browsers may just interpret 429 as equivalent to 400 because it should never happen in the first place.